model/config.go

```diff
@@ -505,35 +511,36 @@ type PluginSettings struct {
 }
 
 type Config struct {
-	ServiceSettings       ServiceSettings
-	TeamSettings          TeamSettings
-	ClientRequirements    ClientRequirements
-	SqlSettings           SqlSettings
-	LogSettings           LogSettings
-	PasswordSettings      PasswordSettings
-	FileSettings          FileSettings
-	EmailSettings         EmailSettings
-	RateLimitSettings     RateLimitSettings
-	PrivacySettings       PrivacySettings
-	SupportSettings       SupportSettings
-	AnnouncementSettings  AnnouncementSettings
-	ThemeSettings         ThemeSettings
-	GitLabSettings        SSOSettings
-	GoogleSettings        SSOSettings
-	Office365Settings     SSOSettings
-	LdapSettings          LdapSettings
-	ComplianceSettings    ComplianceSettings
-	LocalizationSettings  LocalizationSettings
-	SamlSettings          SamlSettings
-	NativeAppSettings     NativeAppSettings
-	ClusterSettings       ClusterSettings
-	MetricsSettings       MetricsSettings
-	AnalyticsSettings     AnalyticsSettings
-	WebrtcSettings        WebrtcSettings
-	ElasticsearchSettings ElasticsearchSettings
-	DataRetentionSettings DataRetentionSettings
-	JobSettings           JobSettings
-	PluginSettings        PluginSettings
+	ServiceSettings       		ServiceSettings
+	TeamSettings          		TeamSettings
+	ClientRequirements    		ClientRequirements
+	SqlSettings           		SqlSettings
+	LogSettings           		LogSettings
+	PasswordSettings      		PasswordSettings
+	FileSettings          		FileSettings
+	EmailSettings         		EmailSettings
+	RateLimitSettings     		RateLimitSettings
+	PrivacySettings       		PrivacySettings
+	SupportSettings       		SupportSettings
+	AnnouncementSettings  		AnnouncementSettings
+	ThemeSettings         		ThemeSettings
+	GitLabSettings        		SSOSettings
+	GoogleSettings        		SSOSettings
+	Office365Settings     		SSOSettings
+	LdapSettings          		LdapSettings
+	ComplianceSettings    		ComplianceSettings
+	LocalizationSettings  		LocalizationSettings
+	SamlSettings          		SamlSettings
+	NativeAppSettings     		NativeAppSettings
+	ClusterSettings       		ClusterSettings
+	MetricsSettings       		MetricsSettings
+	AnalyticsSettings     		AnalyticsSettings
+	WebrtcSettings        		WebrtcSettings
+	ElasticsearchSettings 		ElasticsearchSettings
+	DataRetentionSettings 		DataRetentionSettings
+	ActianceDataExportSettings 	ActianceDataExportSettings
```

Comment:
Only added this field. The rest of the changes in this struct are whitespace only